modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java

```diff
@@ -189,21 +190,16 @@ public void processOpts() {
             setPackageUrl((String) additionalProperties.get(PACKAGE_URL));
         }
 
-        String swaggerFolder = packageName;
-
-        modelPackage = swaggerFolder + File.separatorChar + "models";
```

Comment:
These were overriding the real values of `modelPackage` and `apiPackage` configurations as specified by user.
Also resulted in not being able to leverage for creating import statements.